src/main/java/org/vaadin/pekka/WysiwygE.java

```diff
@@ -135,25 +131,25 @@ protected void initToolbar() {
                 .setAttribute("h6", true);
         getElement().appendChild(headingElement);
         getElement().appendChild(new Element("wysiwyg-tool-blockquote"));
+
+        getElement().addPropertyChangeListener(getClientValuePropertyName(), e-> {
+        });
+        getElement().addEventListener("blur", event -> {
+            setModelValue(getText(), true);
+        });
     }
 
     @Override
-    public void setValue(String value) {
-        getElement().setProperty(getClientValuePropertyName(), value);
+    protected void setPresentationValue(String newPresentationValue) {
+        getElement().setProperty(getClientValuePropertyName(), newPresentationValue);
     }
 
-    @Synchronize("blur")
-    @Override
-    public String getValue() {
+    @Synchronize(value = "blur", property = "value")
+    public String getText() {
```

Comment:
I'd prefer the name of the value method to stay as _value_.

Reply:
getValue is still there, but as it is implemented bit differently, the synchronize annotation don't seem to work anymore.